tests/ui/abi/riscv-discoverability-guidance.rs

```diff
@@ -1,18 +1,17 @@
 // ignore-tidy-linelength
+//@ add-core-stubs
 //@ revisions: riscv32 riscv64
 //
 //@ [riscv32] needs-llvm-components: riscv
 //@ [riscv32] compile-flags: --target=riscv32i-unknown-none-elf -C target-feature=-unaligned-scalar-mem --crate-type=rlib
 //@ [riscv64] needs-llvm-components: riscv
 //@ [riscv64] compile-flags: --target=riscv64gc-unknown-none-elf -C target-feature=-unaligned-scalar-mem --crate-type=rlib
 #![no_core]
-#![feature(
-    no_core,
-    lang_items,
-    abi_riscv_interrupt
-)]
-#[lang = "sized"]
-trait Sized {}
+#![feature(no_core, lang_items, abi_riscv_interrupt)]
+#![feature(trait_alias)] // `cfg(bootstrap)`: Remove this once `MetaSized_` has been removed
```

Comment:
Question: is `trait_alias` needed? I can't find any mentions of `MetaSized_` 🤔

As in, this is a one new `#![feature(..)]` that wasn't present in the original version of this test.

Reply:
Ah, this is just a leftover from pulling this out of my other patch (and that particular comment is leftover from an older version of the patch at that).